README.md

```diff
@@ -8,43 +8,48 @@
 [![License: MIT](https://img.shields.io/badge/License-MIT-blue.svg)](https://opensource.org/licenses/MIT)
 [![Twitter Follow](https://img.shields.io/twitter/follow/operation_code.svg?style=social&label=Follow&style=social)](https://twitter.com/operation_code)
 
-# Temporary Note
+## Temporary Note
 
-This project is not ready for deployment and the documentation may be incorrect.
+  Changes are made on a daily basis to documentation and files. Documentation may not be accurate due to the rapid changes that are occuring. In its current state, this repo is not ready for deployment.
 
-# Welcome!
+## Welcome!
 
 This is incoming v2 front-end application for [OperationCode](https://operationcode.org). We highly recommend [joining our organization](https://operationcode.org/join) to receive an invite to our Slack team. From there, you'll want to join the `#oc-projects` channel. You can get help from multiple professional developers, including people who have worked on the application since day 1!
 
-Before contributing, please review our [GitHub Workflow]().
+Before contributing, please review our [Contributing Guide](CONTRIBUTING.md)
 
 ## Maintainers
 
 For information about the maintainers of the project, check out [MAINTAINERS.md](MAINTAINERS.md).
 
 ## Quick Start
-
 If you're unsure of how to start this app or code for it, don't worry! You're our target audience!
 Please read our [Contributing Guide](CONTRIBUTING.md) to learn everything you need to be able to ask the right questions on our Slack team.
 
-If you're no stranger to React applications and the JavaScript ecosystem:
+**By working in this repo, it is assumed that you are no stranger to React applications, the JavaScript ecosystem, and standard Github workflows such as forking, cloning, & branching.**
```

Comment:
I'd rather see:

In continuing with the quick start instructions, it is assumed that you are no stranger to React applications, the JavaScript ecosystem, and standard GitHub workflows such as forking, cloning, and branching.